onnx/backend/test/runner/__init__.py

```diff
@@ -155,6 +155,36 @@ def _assert_similar_outputs(ref_outputs, outputs):  # type: (Sequence[Any], Sequ
                 rtol=1e-3,
                 atol=1e-7)
 
+    def _retry_excute(self, func, times=3, **kwargs):  # type: (Callable[..., Any], int, **Any) -> Any
```

Comment:
you can still support both `*args` and `**kwargs`:
```python
def _retry_execute(self, func, *args, **kwargs):
   times = kwargs.pop("times", 3)
   ...
```
What I normally would do is to separate the special arguments in a decorator:
```python
def retry(times):
    assert times >= 1
    def wrapper(func):
        @functools.wraps(func)
        def wrapped(*args, **kwargs):
            ...
        return wrapped
    return wrapper
```
and of course then the call side becomes:

```python
@retry(3)
def download_model(...):
    ...

def _prepare_model_data(...):
    ...
    self.download_model(...)
    ...
```